test/e2e/fixture-project/fixtures/simple.ts

```diff
@@ -9,5 +9,11 @@ const PostsQuery = gql`
   }
 `;
 
+const Regression190 = gql`
```

Comment:
Does this also fail for graphql() function invocations?

Reply:
It does, yeah. Want me to add that to the fixture as well? I pushed a fix btw!

Reply:
Would love a fixture in the tada one as we're slowly moving more towards the function invocation

Reply:
> Would love a fixture in the tada one as we're slowly moving more towards the function invocation

Sure! One sec

Reply:
Added!